src/OrchardCore.Modules/OrchardCore.ContentFields/Views/ContentPickerField.Edit.cshtml

```diff
@@ -15,12 +15,12 @@
 <script asp-name="vue-multiselect-wrapper" asp-src="~/OrchardCore.ContentFields/Scripts/vue-multiselect-wrapper.js" at="Foot" depends-on="vuejs, vue-multiselect, sortable, vuedraggable"></script>
 <style asp-name="vue-multiselect-wrapper" asp-src="~/OrchardCore.ContentFields/Styles/vue-multiselect-wrapper.min.css" debug-src="~/OrchardCore.ContentFields/Styles/vue-multiselect-wrapper.css" depends-on="vue-multiselect"></style>
 
-<label asp-for="ContentItemIds">@Model.PartFieldDefinition.DisplayName()</label>
+<div class="mb-3 field-wrapper @fieldName-wrapper" id="@Html.IdFor(x => x.ContentItemIds)_FieldWrapper">
```

Comment:
field-wrapper-@fieldname.Classify()

Reply:
@sebastienros done! Here is an example of generated html code
```
<div class="mb-3 field-wrapper field-wrapper-switch-bool" id="TestField_SwitchBool_Value_FieldWrapper">
    <div class="form-check form-switch">
        <input type="checkbox" class="form-check-input content-preview-select" data-val="true" data-val-required="The Value field is required." id="TestField_SwitchBool_Value" name="TestField.SwitchBool.Value" value="true">
        <label class="form-check-label" for="TestField_SwitchBool_Value">Switch Bool</label>
    </div>
</div>
```